tf-modules/ingest/lzards-backup-task.tf

```diff
@@ -0,0 +1,109 @@
+resource "aws_lambda_function" "lzards_backup_task" {
+  function_name    = "${var.prefix}-LzardsBackup"
+  filename         = "${path.module}/../../tasks/lzards-backup/dist/webpack/lambda.zip"
+  source_code_hash = filebase64sha256("${path.module}/../../tasks/lzards-backup/dist/webpack/lambda.zip")
+  handler          = "index.handler"
+  role             = var.lambda_processing_role_arn
+  runtime          = "nodejs12.x"
+  timeout          = 900
+  memory_size      = 512
+
+  layers = [var.cumulus_message_adapter_lambda_layer_version_arn]
+
+  environment {
+    variables = {
+      CMR_ENVIRONMENT                  = var.cmr_environment
+      stackName                        = var.prefix
+      CUMULUS_MESSAGE_ADAPTER_DIR      = "/opt/"
+      system_bucket                    = var.system_bucket
+      launchpad_passphrase_secret_name = length(var.lzards_launchpad_passphrase) == 0 ? null : aws_secretsmanager_secret.lzards_launchpad_passphrase.name
+      launchpad_certificate            = var.lzards_launchpad_certificate
+      launchpad_api	                   = var.launchpad_api
+      backup_role_arn                  = aws_iam_role.lambda_backup_role.arn
+      lzards_api                       = var.lzards_api
+      provider                         = var.lzards_provider
```

Comment:
Nit: `provider` feels a bit dangerously vague compared to `lzards_provider`

Reply:
updated to lzards_provider